quick_start_cli.md

```diff
@@ -0,0 +1,61 @@
+# Quick Start
+## Setup
+Compatible with Python versions 3.8.10 and higher
+### Install dependencies
+```bash
+# Create an virtual environment 
+# (optional but recommended)
+$ python -m venv venv 
+$ source venv/bin/activate
+# install dependencies
+(venv) $ pip install -r requirements_quick_start.txt
```

Comment:
great, is this requirements_quick_start a cleaned-up version of the reqs?
Would it make sense to keep only one requirements file for the project?

Reply:
it would force us to start thinking about deployment too

Reply:
requirements_quick_start is a cleaned-up version of the reqs for dindex_builder, dindex_store, qbe and ver cli. There would be more reqs after view-distillation and view-presentation are integrated to the quick start.